wandb/data_types.py

```diff
@@ -34,12 +34,18 @@
 from wandb.util import has_num
 from wandb.compat import tempfile
 
+_PY3 = sys.version_info.major == 3 and sys.version_info.minor >= 6
+
+if _PY3:
+    from urllib.parse import quote_plus
+else:
+    from urllib import quote_plus
+
```

Comment:
use six.moves

Reply:
nice... I will make a habit of checking six from here forth... this is second time that six had a better alternative. Thanks for the catch. Incoming commit with changes.